src/wx/xrc/SpeedupConfig.xrc

```diff
@@ -81,7 +87,7 @@
       </object>
       <object class="sizeritem">
         <object class="wxStaticText">
-          <label>Speedup Frame Skip</label>
+          <label>Speedup Throttle (WARNING: May not work if VSYNC is enabled.)</label>
```

Comment:
It is not showing the entire text. It cuts on VSYNC.

Reply:
I see it on wx3-gtk2, I will test on other configs. May need some formatting options for the label in the XML.

Reply:
Actually, I will just implement your solution, then we don't even need the warning.